src/privateer/yacron.py

```diff
@@ -1,7 +1,7 @@
 import os
 import tempfile
 
-import yacron.config
+import yacron.config  # type: ignore
```

Comment:
nb This requires the module `pwd` which is unix-specific

Reply:
The interactions with docker are probably also pretty unix-specific too. But this would be easy enough to remove if we put the validation into the image and run the validator there, I think